docs/configuration.md

```diff
@@ -1,21 +1,143 @@
 # Configuration
 
-Watchmaker is configured using a YAML file. To understand YAML, read up on it
-[here][0].
+Watchmaker is configured using a YAML file. To understand YAML, read up on it [here][0].
 
-Watchmaker comes with a default [config.yaml][1] file, which should work
-out-of-the-box. You can also use it as an example to create your own
-configuration file.
+Watchmaker comes with a default [config.yaml][1] file, which should work out-of-the-box. You can also use it as an example to create your own configuration file.
 
-The configuration is a dictionary. The parent nodes (keys) are `all`, `linux`,
-or `windows`. The parent nodes contain a list of workers to execute, and each
-worker contains parameters specific to that worker. The `all` node is applied
-to every system, and `linux` and `windows` are applied only to their respective
-systems.
+The configuration is a dictionary. The parent nodes (keys) are `all`, `linux`, or `windows`. The parent nodes contain a list of workers to execute, and each worker contains parameters specific to that worker. The `all` node is applied to every system, and `linux` and `windows` are applied only to their respective systems.
 
-You can create a file using the above format with your own set of standard
-values and use that file for Watchmaker. Pass the CLI parameter `--config` to
-point to that file.
+You can create a file using the above format with your own set of standard values and use that file for Watchmaker. Pass the CLI parameter `--config` to point to that file.
 
 [0]: http://www.yaml.org/spec/1.2/spec.html
 [1]: https://github.com/plus3it/watchmaker/blob/develop/src/watchmaker/static/config.yaml
+
+# Config.yaml
+## All
+
+Section for configurations that affect the deployment of the system. All of the configuration parameters can be set in this section. The `all:` section will override parameters that are set in the OS specific sections of the config.yaml, therefor it is suggested practice to break them into the OS blocks of the config. This practice is used in the example config supplied in the next section.
+
+
+### Salt
+
+The salt user defined parameters.
+
+|Option | Type | Description|
+|-------|:-----:|------------|
+|admin_groups: | (list) | The group(s) that you would like the admin accounts placed.|
```

Comment:
Changed to "The group(s) that you would like the admin accounts placed 'in'." (Or 'within' would be acceptable.)